templates/init/functions/typescript/package.lint.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "functions",
   "scripts": {
-    "lint": "eslint \"src/**/*\"",
+    "lint": "eslint --ext .js,.ts .",
```

Comment:
Will this cause eslint to lint compiled js files? I think we don't want that.

Reply:
@joehan I don't think it would, as JS files (apart from `.eslintrc.js`) are listed in the [`.gitignore`](https://github.com/firebase/firebase-tools/tree/bk-2913/templates/init/functions/typescript/_gitignore)

Reply:
(I've been working on coming back to this but @hithomasmorelli is correct)